daemon/cmd/daemon_main.go

```diff
@@ -933,6 +933,10 @@ func init() {
 
 	flags.Bool(option.EnableCustomCallsName, false, "Enable tail call hooks for custom eBPF programs")
 	option.BindEnv(option.EnableCustomCallsName)
+	
+	flags.Bool(option.EnableEgressRevnatAtHostOnly, false,
+		"Enable reverse NAT at node's host interface for egress traffic.")
+	option.BindEnv(option.EnableEgressRevnatAtHostOnly)
```

Comment:
(see the gofmt GH action issue)

Reply:
There is doc bits missing, when do we recommend users to use this agent knob?

Reply:
Updated the doc. Updated to documentation to reflect the need of flag.